### DISCUSSION TITLE
a crash occurred when open gui

### DISCUSSION THREAD

**Discussion by inv1s10n** [The-OpenROAD-Project/OpenROAD#6080] [Q&A] [answered]
![Snipaste_2024-11-01_12-30-04](https://github.com/user-attachments/assets/7ea8176d-f4dd-49fe-aa43-5a6bfcd863a1)

openRoad can run normally on cmd , but gui can not open.

need help pls.

**Comment by maliberty** [comment]
Please update and try again.  An issue was fixed yesterday.

**Reply by gadfort** [reply]
This looks like the same error I get when I accidentally try to open the GUI on a headless system.

**Reply by inv1s10n** [reply, marked as answer]
I have fix this.  Docker container should be assigned to your machine display port by hand.

**Reply by maliberty** [reply]
util/docker_shell will set up the display
